app/scripts/snaps/preinstalled-snaps.ts

```diff
@@ -1,8 +1,10 @@
 import type { PreinstalledSnap } from '@metamask/snaps-controllers';
 import MessageSigningSnap from '@metamask/message-signing-snap/dist/preinstalled-snap.json';
+import AccountWatcherSnap from '@metamask/account-watcher/dist/preinstalled-snap.json';
```

Comment:
Same questions than for `getSnapsList`, if we plan to have this new Snap as a pre-installed one from now on, I wonder if this should be in a separate PR.

(This is really an open-question, and maybe this was the plan to have it all bundle together 😄)

Reply:
I would like to add the account watcher here because we can also use it to test the `accountNameSuggestion` once we tackle that, but I can remove the changes to the selector and add to disable it in the manual testing steps